docs/getting_started/run_smrf.rst

```diff
@@ -30,7 +30,7 @@ below is the function :mod:`run_smrf <smrf.framework.model_framework.run_smrf>`.
         s.loadTopo()
 
         # initialize the distribution
-        s.initializeDistribution()
+        s.create_distribution()
```

Comment:
This will break AWSM. Should be included in [AWSM #68](https://github.com/USDA-ARS-NWRC/awsm/issues/68)